Rx/v2/src/rxcpp/rx-operators.hpp

```diff
@@ -97,8 +97,6 @@ class operator_factory
 
 #include "operators/rx-connect_forever.hpp"
 #include "operators/rx-lift.hpp"
-#include "operators/rx-multicast.hpp"
-#include "operators/rx-publish.hpp"
```

Comment:
If added back, there will be a compile error:
```
rx-multicast.hpp:93:30: error: ‘multicast_tag’ was not declared in this scope
```
since the multicast_tag is used inside of the included file, but defined below these lines..
I can try to put them to the end of the file after all of the `struct YYY_tag {` definitions.